### ISSUE TITLE
How to override the values in the chart ?

### ISSUE THREAD

**Issue by shettyh** [operator-framework/helm-app-operator-kit#35]
Tried adding the values in the spec, like this 

```
apiVersion: xyz.com/v1alpha1
kind: Xyz
metadata:
  name: xyz.com
  labels:
    app: xyz
spec:
  size: 1
  global:
    host: x.x.x.x
    serviceA:
      enabled: false
```

But the chart is not using those values when installed. 

Can anyone please tell me how to properly set the values for the chart with `CR` ?

Thanks in advance

**Comment by alecmerdler** [comment]
I suspect that this is an issue with nested values in the `spec` block not being correctly applied to the Chart, but let me confirm and get back to you. Thanks for bringing this up!

**Comment by shettyh** [comment]
@alecmerdler 
What i observed is it is not even considering the values file properly or something similar issue. It is not creating any services

#36 i added a sample chart with that you can reproduce these issues

**Comment by shettyh** [comment]
Values are getting overridden. But the requirements file was not processed. Fixed the issue in #37
